projects/distributed-tracing/src/shared/dashboard/widgets/table/table-widget-column.model.ts

```diff
@@ -37,9 +45,9 @@ export class TableWidgetColumnModel {
   @ModelProperty({
     key: 'titleTooltip',
     displayName: 'Title Tooltip',
-    type: STRING_PROPERTY.type
+    type: UNKNOWN_PROPERTY.type
   })
-  public titleTooltip?: string;
+  public titleTooltip?: string | TemplateRef<unknown>;
```

Comment:
The api change makes sense for programmatic usages, but the model will never be a template (otherwise we're misusing it)